Firebase/Auth/Source/FIRAuthWebViewController.h

```diff
@@ -45,6 +48,9 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/** @class FIRAuthWebView
```

Comment:
Should this be `FIRAuthWebViewController`?

Reply:
yes it should. Fixed.